user-guide/Advanced_Modules/Dashboards_app/Dashboard_Gateway_installation.md

```diff
@@ -67,3 +72,100 @@ There are two main reasons to consider a Dashboard Gateway setup:
 
    - *connectionString*: The hostname or IP address of the DataMiner Agent to which the Dashboard Gateway has to connect.
    - *connectionUser* and *connectionPassword*: The DataMiner user account that the Dashboard Gateway has to use to connect to the DataMiner Agent (username and password).
+
+## Reverse proxy
+
+Opening a dashboard or low-code app via the Gateway server will cause [Web components](xref:DashboardWeb) displaying a DataMiner Map or video thumbnail to fail to load.
+
+A reverse proxy should be used to give access to these web pages on the DMA via the Gateway server. The reverse proxy will forward any web requests for these web pages to the DMA.
+
+> [!NOTE]
+>
+> - When embedding the URLs in dashboards, make use of relative URLs such as `/maps/maps.aspx?config=yourmapconfig`. This way, the content will both be shown on a dashboard accessed via the Gateway server and a dashboard accessed locally via the DMA.
+> - No reverse proxy is needed for DataMiner Maps that are displayed using a [Generic map component](xref:DashboardGenericMap).
```

Comment:
Only correct for the new maps component (only available from 10.3.2 onwards, with the ReportsAndDashboardsGQIMaps soft launch flag enabled). The old maps component requires the reverse proxy.